layouts/wrapper.html

```diff
@@ -33,12 +33,20 @@
 		{{/outputBlock}}
 	</head>
 	<body class="o-hoverable-on">
+
+		{{#if showSkipLinks}}
+			<a class="u-visually-hidden" href="#primary-nav">Skip to navigation</a>
+			<a class="u-visually-hidden" href="#content">Skip to content</a>
```

Comment:
Have changed. But still don't like the fact we need to add to each nightwatch test suite and remember to add it. Where are these magical **pre-launch** check lists you speak of?